charts/jindofs/templates/fuse/daemonset.yaml

```diff
@@ -134,7 +134,7 @@ spec:
           {{- end }}
       restartPolicy: Always
       {{- if $notEnableDnsConfig }}
-      dnsPolicy: ClusterFirstWithHostNet
+      dnsPolicy: {{ .Values.useHostNetwork | ternary "ClusterFirstWithHostNet" "ClusterFirst" }}
```

Comment:
I think there is no need to set fuse in container network, since it has no port.

Reply:
ok，prepared for client dist cache used

Reply:
Is it possible to put two fuses in the same node? How to handle the mount point?